{
    "url": "https://api.github.com/repos/ethereumclassic/ECIPs/issues/comments/674831352",
    "html_url": "https://github.com/ethereumclassic/ECIPs/issues/333#issuecomment-674831352",
    "issue_url": "https://api.github.com/repos/ethereumclassic/ECIPs/issues/333",
    "id": 674831352,
    "node_id": "MDEyOklzc3VlQ29tbWVudDY3NDgzMTM1Mg==",
    "user": {
      "login": "q9f",
      "id": 58883403,
      "node_id": "MDQ6VXNlcjU4ODgzNDAz",
      "avatar_url": "https://avatars1.githubusercontent.com/u/58883403?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/q9f",
      "html_url": "https://github.com/q9f",
      "followers_url": "https://api.github.com/users/q9f/followers",
      "following_url": "https://api.github.com/users/q9f/following{/other_user}",
      "gists_url": "https://api.github.com/users/q9f/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/q9f/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/q9f/subscriptions",
      "organizations_url": "https://api.github.com/users/q9f/orgs",
      "repos_url": "https://api.github.com/users/q9f/repos",
      "events_url": "https://api.github.com/users/q9f/events{/privacy}",
      "received_events_url": "https://api.github.com/users/q9f/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2020-08-17T11:41:47Z",
    "updated_at": "2020-08-17T11:41:47Z",
    "author_association": "MEMBER",
    "body": "Also, some historical context can be found in #174 #187 and #194 - back then Wei missed to attend a call and blocked all further work on that. But maybe we can recover some of the work and discussion. See #339 ",
    "reactions": {
      "url": "https://api.github.com/repos/ethereumclassic/ECIPs/issues/comments/674831352/reactions",
      "total_count": 0,
      "+1": 0,
      "-1": 0,
      "laugh": 0,
      "hooray": 0,
      "confused": 0,
      "heart": 0,
      "rocket": 0,
      "eyes": 0
    },
    "performed_via_github_app": null
  }